src/block_service/fds/fds_service.cpp

```diff
@@ -37,9 +37,11 @@ DSN_DEFINE_uint64("pegasus.server",
                   64 << 20,
                   "rocksdb options.write_buffer_size");
 
-DSN_DEFINE_uint32("replication", fds_write_limit_rate, 100, "rate limit of fds(MB/s)");
+DSN_DEFINE_uint32("replication", fds_write_limit_rate, 100, "write rate limit of fds(MB/s)");
 
-DSN_DEFINE_uint32("replication", fds_read_limit_rate, 100, "rate limit of fds(MB/s)");
+DSN_DEFINE_uint32("replication", fds_read_limit_rate, 100, "read rate limit of fds(MB/s)");
+
+DSN_DEFINE_uint32("replication", fds_read_batch_size, 5, "read batch size of fds(MB)");
```

Comment:
Have you test the default 5 is OK？

Reply:
I set it to 100MB to ensure the speed will be very quick.  Somebody can change it to a small number when he wants to smooth the network speed.